app/views/styleguide/components/_slider.html.erb

```diff
@@ -0,0 +1,6 @@
+<%= render 'components/slider', { option: 'leave-pot-untouched', initial_value: rand(0..150) } %>
```

Comment:
Maybe include an example for the default slider too? (i.e. with no `option` key defined)